src/agenda.rs

```diff
@@ -460,7 +460,7 @@ pub fn prioritization<'a>() -> Box<dyn Action> {
     });
 
     queries.push(QueryMap {
-        name: "i_nominated_t_compiler",
+        name: "nominated_t_compiler",
```

Comment:
unrelated, I've renamed all `i-nominated_*` to `nominated_*` to have all the same label